app/heron/factories/concerns/contents.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+module Heron
+  module Factories
+    module Concerns
+      #
+      # This module adds validation and tools to manage factories that act as content in a receptacle
+      # Eg: ::Heron::Factories::Sample
+      #
+      # **Requirements**
+      # - The method or attribute content_factory needs to be defined with the class that act as
+      #   factory for contents. Eg: ::Heron::Factories::Sample
+      # - The content configuration should be already stored in @params
+      # - The content configuration object needs to be in @params[recipients_key]
+      # - The method or attribute recipients_key will need to identify the key where the config is
+      #   stored. Eg: :wells
```

Comment:
```recipients_key``` could be set to :recipients mandatorily so tube racks and plates use it. Could be change optionally if ever needed.

Reply:
Not needed. Makes more sense to name it wells for plates and tubes for racks